WcaOnRails/app/assets/javascripts/selectize.tags_options.js

```diff
@@ -0,0 +1,14 @@
+function defaultSelectizeOptions(options) {
```

Comment:
The point of the `wca` object is to try to avoid cluttering the global javascript namespace. Could you throw this into `wca.defaultSelectizeOptions` instead?

Reply:
Sure, and more generally I'll clean the whole javascript in this PR, I rushed it!